test/onnx/test_utility_funs.py

```diff
@@ -38,6 +38,9 @@ def forward(self, x):
                 return b + x
 
         _set_opset_version(9)
+        import torch.onnx.symbolic_registry as sym_registry
+        sym_registry.register_version('', 9)
```

Comment:
what's the purpose of calling register_version('', 9) here?

Reply:
here _set_opset_version() and _model_to_graph() are called instead of _export(), so register_version() is not called.
I will try moving the call to register_version() to avoid this.